pkg/plugin/provision/spirehelm/spirehelm_test.go

```diff
@@ -112,6 +142,24 @@ func TestSpireHelm_TearDown(t *testing.T) {
 	assert.EqualExportedValues(t, want, statuses)
 }
 
+func TestSpireHelm_TearDown_specificTrustZone(t *testing.T) {
+	providerFactory := newFakeHelmSPIREProviderFactory()
+	spireAPIFactory := newFakeSPIREAPIFactory()
+	spireHelm := NewSpireHelm(providerFactory, spireAPIFactory)
+	ds := newFakeDataSource(t, defaultConfig())
+
+	opts := provision.TearDownOpts{KubeCfgFile: "fake-kube.cfg", TrustZones: []string{"tz1"}}
+	statusCh, err := spireHelm.TearDown(context.Background(), ds, &opts)
+	require.NoError(t, err, err)
+
+	statuses := collectStatuses(statusCh)
+	want := []*provisionpb.Status{
+		provision.StatusOk("Uninstalling", "Uninstalling SPIRE chart for local1 in tz1"),
+		provision.StatusDone("Uninstalled", "Uninstallation completed for local1 in tz1"),
+	}
```

Comment:
Does `collectStatuses` encompass the entire output of the a given test scenario of a `down` run here? i.e. can we infer that tz1 and only tz1 was uninstalled from this test?

Could add a `tz2` that we ensure is not touched in those statuses for this unit test